software/forgejo.yml

```
@@ -0,0 +1,16 @@
+name: Forgejo
+website_url: https://forgejo.org
+source_code_url: https://codeberg.org/forgejo/forgejo/
+description: A lightweight software forge maintained by an inclusive community under the umbrella of the non-profit Codeberg e.V. and focused on scaling, federation, and privacy.
```

```suggestion
description: A lightweight software forge focused on scaling, federation, and privacy (fork of Gitea).
```

Let's keep it concise and to the point ;)
Agreed with your other comments.
Thanks

---

Sure, I've made that change. Thanks @nodiscc .